### PR TITLE
F post test

### PR DESCRIPTION
**Pull request by MarieEngel**
Can you have a. look at my tests? :)

### CODE REVIEW

File: tests/test_post.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        )
+        response = self.client.get("/")
+        self.assertTrue("Version 2" in str(response.content))
+
+
+    def test_home_view_logged_out(self):
+        """Tests that logged out users can't see posts."""
+        response = self.client.get("/")
+        self.assertEqual(response.status_code, 200)
+        self.assertNotContains(response, "Latest Posts")
```
Comment (SyncopatedPandemonium) [inline tests/test_post.py:81]: You can't retrieve body when the response is 302.
We can check here if unauthorised user is redirected to login page instead.

**Review by SyncopatedPandemonium — CHANGES_REQUESTED** [review]
Please check the comment to tests/test_post.py file line 81.

File: tests/test_post.py (hunk 1 above)
Comment (MarieEngel) [reply tests/test_post.py:81]: Done :)

Comment (SyncopatedPandemonium) [reply tests/test_post.py:81]: Well done

**Review by SyncopatedPandemonium — APPROVED** [review]

**Pull request merged**